bids-validator/utils/type.js

```diff
@@ -311,8 +306,7 @@ export default {
         this.isIEEG(path) ||
         this.isBehavioral(path) ||
         this.isFuncBold(path) ||
-        this.isPETData(path) ||
-        this.isPETBloodData(path)
+        this.isPETData(path)
```

Comment:
Since we're looking at consolidating the rules for "pet" and "pet_blood", and we still want to collect that modality information, we should keep `isPET` and `isPETBlood` here